core/src/test/scala/org/apache/spark/deploy/rest/StandaloneRestSubmitSuite.scala

```diff
@@ -83,6 +83,26 @@ class StandaloneRestSubmitSuite extends SparkFunSuite with BeforeAndAfterEach {
     assert(submitResponse.success)
   }
 
+  test("create submission with multiple masters") {
+    val submittedDriverId = "your-driver-id"
+    val submitMessage = "my driver is submitted"
+    val masterUrl = startDummyServer(submitId = submittedDriverId, submitMessage = submitMessage)
+    val conf = new SparkConf(loadDefaults = false)
+    val RANDOM_PORT = 9000
+    val allMasters = "%s,%s:%d".format(masterUrl, Utils.localHostName(), RANDOM_PORT)
```

Comment:
Can we use interpolation here? Below, I'm not sure anything tests that the new master config takes hold?

Reply:
Ping @bsikander

Reply:
@srowen I was away for a few days. Will respond back soon.